### DISCUSSION TITLE
Increasing the simulation duration for 5MW_Land_DLL_WTurb

### DISCUSSION THREAD

**Discussion by clbanjai** [OpenFAST/openfast#2300] [Q&A]
Hello,

I have a similar problem to #832, and I tried following the steps and changed the usable time in https://github.com/OpenFAST/r-test/blob/main/glue-codes/openfast/5MW_Baseline/Wind/90m_12mps_twr.inp from 60 to 630, but what I am stuck at is the rerunning TurbSim. I downloaded the executable turbsim file, and I tried running it with 90m_12mps_twr.inp as the input and it asked me to unzip a bunch of files. I tried rerunning the simulation and it aborted at the 62 second mark.

**Comment by jjonkman** [comment]
Hi @clbanjai,

Can you share how you calling TurbSim and what error it is reporting when you run it?

Best regards,
